### PR TITLE
Landing page 

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

**Comment by elbotho** [comment]
fixes #52

File: src/components/landing/landingcsshelper.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { css } from 'styled-components'
+import { lighten } from 'polished'
+
+export const makeSVGStyle = ({ Parent }) => {
```
Comment (Entkenntnis) [inline src/components/landing/landingcsshelper.ts:4]: all files in `components`-folder are expected to be components. CSS-Helper goes into helper, but in this case, the helper is tied to a component and shouldn't be exposed.

File: .babelrc [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["next/babel"],
-  "plugins": [["styled-components", { "ssr": true }]]
+  "plugins": [["styled-components", { "ssr": true }], "inline-react-svg"]
```
[the remark this answers is not in the document]
Comment (elbotho) [reply .babelrc]: I tried 4 solutions and had some kind of problem/drawback with all of them. So I went with the one where I could find the best tutorial basically ;)

**Comment by elbotho** [comment]
Thanks for the cleanup 👍

**Comment by Entkenntnis** [comment]
If nothing is broken I would merge this one.

**Comment by elbotho** [comment]
The white text sections are broken (flex problem) on bigger screens.
("Serlo.org ist die Wikipedia fürs Lernen" and "Werden Sie Teil unserer Bewegung für freie Bildung
"

**Comment by elbotho** [comment]
The h2 titles need `width: 100%` somehow that got lost I think.

**Comment by elbotho** [comment]
and maybe `margin-left: 0` on the `ul` than im happy :)

**Comment by Entkenntnis** [comment]
Let's merge this one :yum:

**Comment by elbotho** [comment]
Nice!

**Pull request merged**